src/icons.js

```diff
@@ -65,7 +65,7 @@ export { default as listOrdered } from '../images/icons/list-ordered.svg';
 export { default as listUnordered } from '../images/icons/list-unordered.svg';
 export { default as list } from '../images/icons/list.svg';
 export { default as lock } from '../images/icons/lock.svg';
-export { default as lockFilled } from '../images/icons/lock-filled.svg';
+export { default as lockFilled } from '../images/icons/lock-alt-filled.svg';
```

Comment:
This module registers icons for the "legacy" coverage in the pattern library. I've left the export name as `lockFilled` so I don't need to go in and update legacy pattern-library pages.